salt/runners/salt.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+'''
+Runner to call execution modules
+
+.. versionadded:: Carbon
+'''
+# import python libs
+from __future__ import absolute_import
+from __future__ import print_function
+import logging
+
+# import salt libs
+from salt.loader import minion_mods
+
+log = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+def cmd(fun, args=None, kwargs=None):
```

Comment:
After trying this out, I am curious why you chose to define the args and kwargs this way, instead of `*args, **kwargs`. It took me a little while of stumbling around to see I wasn't looking at your example closely enough and was intuitively trying to do:
```
salt-run salt.cmd test.sleep 30
```
And ran into a stack trace. It worked when I switched in to your syntax, but this took several tries.

I suspect you may have chose this approach because you may have been hitting something like this:
```
root@rallytime:~/SaltStack/salt# salt-run salt.cmd test.sleep 3
Exception occurred in runner salt.cmd: Traceback (most recent call last):
  File "/root/SaltStack/salt/salt/client/mixins.py", line 364, in _low
    data['return'] = self.functions[fun](*args, **kwargs)
  File "/root/SaltStack/salt/salt/runners/salt.py", line 35, in cmd
    return minion_mods(__opts__).get(fun)(*args, **kwargs)  # pylint: disable=undefined-variable
TypeError: sleep() got an unexpected keyword argument '__pub_master_job_cache'
[INFO    ] Runner completed: 20160610232923688071
```

I ran into this issue also, and had to work around it by filtering out those `__pub` keys when trying to use the `wheel.cmd` function in the `saltutil` module: https://github.com/saltstack/salt/pull/33414/files#diff-a72c3e04b3df0479b574ff06020607efR1028